tests/fixtures/phase3_fixtures.py

```
@@ -0,0 +1,408 @@
+- Performance measurement utilities
+- Memory monitoring helpers
+- Concurrent testing utilities
+"""
+
+import asyncio
+import gc
+import os
+import tempfile
+import time
+from pathlib import Path
+from typing import Dict, List, Optional
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)